.gitignore

```diff
@@ -2,7 +2,6 @@
 .settings
 .project
 *.sh
-*.bat
```

Comment:
You could force version the bat file, even without loosing this line

Reply:
Like so: `git add --force file.foo`
But why would you introduce such ambiguity? One will see .bat file in the repo and just commit without reviewing, then their .bat is gone on the next checkout.  If there are .bat files to be excluded from the PR, its part of the review to ask the author to remove them (or do it yourself)?